src/Components/CreateImageWizard/utilities/requestMapper.ts

```diff
@@ -194,6 +195,27 @@ function commonRequestToState(
       blueprintName: request.name || '',
       blueprintDescription: request.description || '',
     },
+    users:
+      request.customizations?.users?.map((user) => ({
+        name: user.name,
+        password: user.password || undefined,
+        ssh_key: user.ssh_key || undefined,
+      })) || [],
+    userInfo: request.customizations?.users?.length
+      ? {
+          userName: request.customizations.users[0].name,
+          password: request.customizations.users[0].password || '',
+          confirmPassword: '',
+          sshKey: request.customizations.users[0].ssh_key || '',
+          administrator: false,
+        }
+      : {
+          userName: '',
+          password: '',
+          confirmPassword: '',
+          sshKey: '',
+          administrator: false,
+        },
```

Comment:
Here we have some duplicity as well. I believe it would be better to either work with a single user, removing the `users` array, or better be prepared for multiple users in the future, removing the `userInfo` and mapping all values into `users[]` using a custom type that extends User type and includes all extra info we need.